doofinder-for-woocommerce/includes/data-feed/class-data-feed-item.php

```diff
@@ -346,6 +349,31 @@ private function add_prices() {
 		}
 	}
 
+	/**
+	 * Add product leader.
+	 * Product with variations and variations themselves require 'df_group_leader' and 'df grouping_id' fields.
+	 * If product has variations, then variations get his id.
+	 */
+	private function add_group_leader() {
+		if ( 'no' === $this->settings['split_variable'] ) {
+			return;
+		}
+
+		$id = $this->post->ID;
+		$product = wc_get_product( $id );
+		$parent = wc_get_product( $product->get_parent_id() ) ?: false;
+
+		if ( $product->is_type( 'variable' ) ) {
+			$this->fields[ 'df_group_leader' ] = 'true';
```

Comment:
@marcinsz84 I think this field should be a boolean, not a string…

Reply:
Fixed